### DISCUSSION TITLE
1.2.4.1启动fe的时候报Invalid maximum heap size: -Xmx8192m错误

### DISCUSSION THREAD

**Discussion by ruanmingxing110** [apache/doris#33611] [RFC]
错误信息：
Invalid maximum heap size: -Xmx8192m
The specified size exceeds the maximum representable size.
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.

jdk版本：
java version "1.8.0_371"
Java(TM) SE Runtime Environment (build 1.8.0_371-b11)
Java HotSpot(TM) Server VM (build 25.371-b11, mixed mode)

**Comment by yagagagaga** [comment]
Upgrade to version 1.2.8 or increase heap memory.
